examples/addsvc/client/main.go

```diff
@@ -108,10 +136,15 @@ func main() {
 		logger.Log("err", "invalid method "+method)
 		os.Exit(1)
 	}
+
+	if len(*lightstepAccessToken) > 0 {
+		lightstep.FlushLightStepTracer(tracer)
+	}
```

Comment:
Can remove this if you add a `defer lightstep.FlushLightStepTracer(tracer)` after the creation in the switch block above.